configure.ac

```diff
@@ -108,6 +108,9 @@ AS_IF([test "x$enable_fontconfig" != xno], [
         fontconfig=true
     ], [
         fontconfig=false
+        AS_IF([test "x$enable_fontconfig" = xyes], [
+            AC_MSG_ERROR([Couldn't locate fontconfig required for requested feature.])
```

Comment:
How about: “fontconfig support was requested, but it was not found.”

Also, the `--help` message claims the default is `--enable-fontconfig=enabled`, which is especially confusing after this change. Set that to `check`?

Also, CoffeeFlux is gonna hate you for changing configure logic yet again :-) But this seems sensible to me.

Reply:
Fixed the help string and changed the failure messages.